### ISSUE TITLE
pigeon_plugin.h 

### ISSUE THREAD

**Issue by mitchross** [gaaclarke/pigeon_plugin_example#1]
What is the significance of pigeon_plugin.h?

**Comment by gaaclarke** [comment]
The way cocoapods does swift-objc interop it may not be required.  It's convention to have a top level import for a plugin that represents everything.

**Comment by mitchross** [comment]
@gaaclarke FWIW , Im working on a swift/pigeon plugin. I have to include a similar file or it will not compile. I get a .h header file not found! 
![Screen_Shot_2020-11-29_at_12 55 59_AM](https://user-images.githubusercontent.com/6330506/100655121-d7578180-3318-11eb-9d38-c18d532bf1b2.png)

I ended up fixing with manually adding "flutter_vlc_player.h"

![files](https://user-images.githubusercontent.com/6330506/100655185-f0f8c900-3318-11eb-99d8-71f7eb340bd9.PNG)

I highly recommend adding these to your example notes. It was not auto generated for me and spent several hours trying to fix this on my end. I was able to figure it out by studying your project closely! Thanks for putting out this sample project, super helpful!
